cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -1149,7 +1167,9 @@ def is_privileged():
     """
     privileged = hookenv.config('allow-privileged').lower()
     if privileged == 'auto':
-        return is_state('kubernetes-master.gpu.enabled')
+        return \
+            is_state('kubernetes-master.gpu.enabled') or \
+            is_state('kubernetes-master.ceph.configured')
```

Comment:
Do we want to trigger this off ceph-storage.available and then remove the force_privileged? Would that work as we intend here?

Reply:
The last commit should resolve that